src/Csv/CsvGenerator.fs

```diff
@@ -28,7 +29,8 @@ module internal CsvTypeBuilder =
       { TypeForTuple = typWithoutMeasure
         Property = ProvidedProperty(propertyName, typ, GetterCode = fun (Singleton row) -> Expr.TupleGet(row, index))
         Convert = fun rowVarExpr -> conv <@ TextConversions.AsString((%%rowVarExpr:string[]).[index]) @>
-        ConvertBack = fun rowVarExpr -> convBack (Expr.TupleGet(rowVarExpr, index)) } )
+        ConvertBack = fun rowVarExpr -> let backExpr = convBack (Expr.TupleGet(rowVarExpr, index)) in <@@ if (%%backExpr : string) = null then "" else (%%backExpr : string) @@>
```

Comment:
The null to "" conversion needs to be done on the row constructor instead, to maintain the invariant of not having null as a string value on CsvProvider anywhere